proco/connection_statistics/api_urls.py

```diff
@@ -6,4 +6,10 @@
 
 urlpatterns = [
     path('global-stat/', api.GlobalStatsAPIView.as_view(), name='global-stat'),
+    path('country/<int:country_id>/daily-stat/', api.CountryDailyStatsListAPIView.as_view(), name='country-daily-stat'),
+    path(
+        'country/<int:country_id>/weekly-stat/<int:week>/<int:year>/',
```

Comment:
please change arguments ordering to be more logical
country/<int:country_id>/weekly-stat/<int:year>/<int:week>/